MCUboot/README.md

```diff
@@ -1,163 +1,52 @@
+```shell
+./scripts/fota.sh -e=mcuboot
+```
+> **Note**: You can build the example without a connected target board. The script indicates the location of the factory firmware and the update binary. The factory firmware would have to be flashed manually when the target board is indeed connected later. The update binary would be used in the demonstration stage.
 
-1. Extract the public key into a C data structure so that it can be built into the bootloader:
+## Demonstration
 
-   ```shell
-   mcuboot/scripts/imgtool.py getpub -k signing-keys.pem >> signing_keys.c
-   ```
-
-1. Build the bootloader:
-
-   ```shell
-   mbed compile -t GCC_ARM -m NRF52840_DK
-   ```
-
-#### Building and signing the primary application
-
-1. Build the primary application:
-
-   ```shell
-   cd ../application && mbed compile -t GCC_ARM -m NRF52840_DK
-   ```
-
-1. Copy the HEX file into the bootloader folder:
-
-   ```shell
-   cp BUILD/NRF52840_DK/GCC_ARM/application.hex ../bootloader && cd ../bootloader
-   ```
-
-1. Sign the initial application using the rsa-2048 keys:
-
-   ```shell
-   mcuboot/scripts/imgtool.py sign -k signing-keys.pem --align 4 -v 0.1.0 --header-size 4096 --pad-header -S 0xC0000 --pad application.hex signed_application.hex
-   ```
-   
-#### Creating and flashing the "factory firmware"
-
-1. Merge the bootloader and signed application:
-
-   ```shell
-   hexmerge.py -o merged.hex --no-start-addr BUILD/NRF52840_DK/GCC_ARM/bootloader.hex signed_application.hex
-   ```
-
-1. Erase the chip:
-   
-   ```shell
-   pyocd erase --chip
-   ```
-   
-1. Flash the "factory firmware" onto the board by copying `merged.hex` over to its mount point:
-
-   ```shell
-   cp merged.hex <mount_point>
-   ```
-   
-   The target's mount point can be found by running `mbedls`.
-   
-#### Creating the update binary
-
-1. Change the application's version number in its [mbed_app.json](target/application/mbed_app.json) to 0.1.1 and rebuild it:
-
-   ```shell
-   cd ../application && mbed compile -t GCC_ARM -m NRF52840_DK
-   ```
-
-1. Copy the HEX file into the bootloader folder:
-
-   ```shell
-   cp BUILD/NRF52840_DK/GCC_ARM/application.hex ../bootloader && cd ../bootloader
-   ```
-
-1. Sign the update application using the rsa-2048 keys:
-
-   ```shell
-   mcuboot/scripts/imgtool.py sign -k signing-keys.pem --align 4 -v 0.1.1 --header-size 4096 --pad-header -S 0x55000 application.hex signed_update.hex
-   ```
-
-1. Generate a raw binary file from `signed_update.hex` so that it can be transported over BLE:
-
-   ```shell
-   arm-none-eabi-objcopy -I ihex -O binary signed_update.hex signed_update.bin
-   ```
-
-### Demonstration
-
-1. Open a serial terminal on the host PC to receive serial prints from the target:
+Please make sure to activate the virtual environment, located in the root folder, before proceeding further. Also, ensure that the target board is connected and has been flashed with the factory firmware from the build stage.
```

Comment:
add a 
   ```shell
  block
   ```
showing how to do that

Reply:
Would this be for the "Please make sure..." part? If yes, I could do a quote section as follows:
> **Important**: Please make sure...

Reply:
You're on holiday. Stop.